examples/drivers/windows/otLwf/filter.h

```diff
@@ -36,8 +36,11 @@
 #define _FILT_H
 
 // The maximum allowed addresses an OpenThread interface
-#define OT_MAX_ADDRESSES 10
-#define OT_MAX_AUTO_ADDRESSES (OT_MAX_ADDRESSES - 4)
+#if OPENTHREAD_ENABLE_DHCP6_CLIENT
```

Comment:
Yes if`otCachedAddr[]` is used to cache all the Unicast Addresses the thread device could have